First build / the examples do not build

When cloning the repo from scratch, the cmake step is fine but when I try to compile the examples, there is a "includedir" issue:

```
...clone...
mkdir build
cd build
cmake ..
[...]
cd examples
make
```

and I get

```
[ 11%] Building CXX object examples/cpp/CMakeFiles/ponca_binding.dir/ponca_binding.cpp.o
In file included from /tmp/ponca/examples/cpp/ponca_binding.cpp:19:
In file included from /tmp/ponca/Ponca/src/Fitting/gls.h:10:
/tmp/ponca/Ponca/src/Fitting/./curvature.h:11:10: fatal error: 'Eigen/Eigenvalues' file not found
#include <Eigen/Eigenvalues>
         ^~~~~~~~~~~~~~~~~~~
1 error generated.
``` 

--- a/CHANGELOG
+++ b/CHANGELOG
@@ -10,7 +10,7 @@ v.0.2
     - [api] New public entry points are the header files <Ponca/Ponca> and <Ponca/Fitting> (#9)
 
 - Buildchain
-    - [cmake] Fix several bugs with Eigen submodule (#17,#19)
+    - [cmake] Fix several bugs with Eigen submodule (#17,#19,#26)
     - [cmake] Allow use as submodule (#15)
     - [cmake] Cmake package is now relocable (#9)
 

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -76,7 +76,7 @@ endif()
 
 macro(ponca_handle_eigen_dependency TARGETNAME)
     ## Should be retrieved by transitive linking, however it seems to fails with older cmake versions
-    if(Eigen3_FOUND AND ${CMAKE_VERSION} VERSION_LESS "3.19")
+    if(Eigen3_FOUND)
         get_target_property(MODULE Fitting INTERFACE_LINK_LIBRARIES )
         if ("${MODULE}" STREQUAL "Eigen3::Eigen")
             target_link_libraries(${TARGETNAME} PUBLIC Eigen3::Eigen)

